### PR TITLE
Fix build configuration for C# app and target framework version of the lib.

### PR DESCRIPTION
**Pull request by rohanp-msft**
If you try to build the C# app as is after cloning the repo, the wapproj won't be built. Deploy won't work either. This is because these things weren't enabled for Any CPU, which is the configuration when you open the solution.

Also, this change updates the target framework version of the lib to match the consuming app. An app dev shouldn't have to get an older version of the SDK unnecessarily.

**How verified**
Built the lib.
Built and ran the app.

### CODE REVIEW

File: MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app.sln [hunk 1]
```diff
@@ -18,6 +18,8 @@ Global
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{28B258CA-5ECC-41EE-A5B0-C2F24F3A4984}.Debug|Any CPU.ActiveCfg = Debug|x86
+		{28B258CA-5ECC-41EE-A5B0-C2F24F3A4984}.Debug|Any CPU.Build.0 = Debug|x86
```
Comment (huichen123) [inline MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app.sln:21]: >Debug [](http://example.com/codeflow?start=41&length=5)

also release

Comment (huichen123) [inline MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app.sln:21]: >x86 [](http://example.com/codeflow?start=71&length=3)

any cpu?

Comment (rohanp-msft) [reply MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app.sln:21]: this was auto generated by Visual Studio

**Review by huichen123 — APPROVED** [review]
:shipit:

**Pull request merged**